src/controllers/view/index.js

```diff
@@ -0,0 +1,16 @@
+const path = require("path");
+
```

Comment:
You can refactor these functions and use an advanced concept of currying your functions. The 2 controller functions are very similar and the only difference is the file name. It's quite a tricky concept to understand, but I'll leave you an article here to read through: https://medium.com/javascript-scene/curry-and-function-composition-2c208d774983 

```

When you reference this function in your `routes` you will need to pass in the file name.
const renderer = (fileName) => (req, res) => {
  const filePath = path.join(__dirname, `../../../public/${fileName}.html`);
  res.sendFile(filePath);
};
```